wheel/chia_rs.pyi

```diff
@@ -87,3 +88,26 @@ class LazyNode:
     def atom() -> bytes: ...
 
 def serialized_length(program: bytes) -> int: ...
+
+class Coin:
+    parent_coin_info: bytes32
+    puzzle_hash: bytes32
```

Comment:
The choice to use `bytes32` here deserves some scrutiny.

1. This adds a dependency onto `chia-blockchain`
2. These types are actually just regular `bytes`, but guaranteed to be 32 bytes.
3. Specifying `bytes` as the type here causes a *lot* of mypy issues in `chia-blockchain`, when (afaik) `bytes32` doesn't have any important behaviour that differs from a `bytes` type of size 32. (as opposed to `uint64`)

Reply:
Have you considered moving `bytes32` to `chia_rs`?

Reply:
it would still be a python type though, right? I'm not sure it's possible to make `bytes32` a rust type and still appear to be derived from the built-in `bytes` type.

So, no, I didn't consider that. It sounds a bit scary.

Reply:
Yeah, you'd probably have to either create a `bytes32` type in rust (that subclasses `bytes`), or do it in python and call out from rust to python to create it. I'd say let's get this in as-is for now and fight with `pyo3` another day.